test/spec/helper.clears.js

```diff
@@ -68,6 +71,58 @@ test( "Clearing the same field from multiple elements should remove it everywher
   t.end();
 } );
 
+test( "Clear with a function : neutral and abosorbing ops", function( t ) {
```

Comment:
Yeah that's not a good and easy to understand. It's like in math, where 0 is absorbant (almost sure it's a wrong translation) for the multiplication...